x-pack/plugins/case/server/services/index.ts

```diff
@@ -75,13 +75,13 @@ interface PatchCasesArgs extends ClientArgs {
 }
 interface UpdateCommentArgs extends ClientArgs {
   commentId: string;
-  updatedAttributes: Partial<CommentAttributes & PushedArgs>;
+  updatedAttributes: Partial<CommentAttributes>;
   version?: string;
 }
 
 interface PatchComment {
   commentId: string;
-  updatedAttributes: Partial<CommentAttributes & PushedArgs>;
+  updatedAttributes: Partial<CommentAttributes>;
```

Comment:
`Pick` instead?

Reply:
I think we can update any CommentAttributes or do i miss something?